wegodang/products/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+class Category(models.Model):
+    name  = models.CharField(max_length=45)
+    image = models.CharField(max_length=200)
+
+    class Meta:
+        db_table = 'catergories'
+
+class Product(models.Model):
+    name           = models.CharField(max_length=45)
+    total_pirce    = models.DecimalField(max_digits=10, decimal_places=2)
+    goal_pirce     = models.DecimalField(max_digits=10, decimal_places=2)
+    suppoters      = models.IntegerField()
+    start_date     = models.DateField()
+    end_date       = models.DateField()
+    story          = models.TextField()
+    slide_title    = models.TextField()
+    slide_subtitle = models.TextField()
+    category_id    = models.ForeignKey(Category, on_delete=models.CASCADE, related_name="Category_Product")
+
+    class Meta:
+        db_table = 'products'
+
+class ProductModel(models.Model):
+    name       = models.CharField(max_length=45)
+    price      = models.DecimalField(max_digits=10, decimal_places=2)
+    content    = models.TextField()
+    product_id = models.ForeignKey(Product, on_delete=models.CASCADE, related_name="Product_ProductModel")
+
+    class Meta:
+        db_table = 'product_models'
+
+class ProductImage(models.Model):
+    product_id = models.ForeignKey(Product, on_delete=models.CASCADE, related_name="Product_ProductImage")
```

Comment:
model에서 foreignkey를 만들 때에는 _id를 빼고 만들어 주세요.

```suggestion
    product = models.ForeignKey(Product, on_delete=models.CASCADE, related_name="product_images")
```

Reply:
수정했습니다!